### PR TITLE
Move Roku.roku_remote_tool 4.0.5.1327 to Roku.RemoteTool 4.0.5.1327

### PR DESCRIPTION
**Pull request by Trenly**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/137923)

### CODE REVIEW

**Comment by stephengillie** [comment]
This package uninstalled itself during manual validation.

**Comment by Trenly** [comment]
> This package uninstalled itself during manual validation.

Interesting. Using `SandboxTest.ps1` I didn't see it uninstall itself at all

File: manifests/r/Roku/RemoteTool/4.0.5/Roku.RemoteTool.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Roku.RemoteTool
+PackageVersion: 4.0.5
+PackageLocale: en-US
+Publisher: Roku
+PackageName: roku_remote_tool
+License: Proprietary
+Copyright: Copyright © 2022 Roku
+ShortDescription: roku_remote_tool
```
Comment (mdanish-kh) [inline manifests/r/Roku/RemoteTool/4.0.5/Roku.RemoteTool.locale.en-US.yaml:11]:
```suggestion
ShortDescription: The Roku Remote Tool provides developers a quick and intuitive way to create reusable scripts for ad-hoc testing of their channel.
Documentations:
- DocumentLabel: Roku platform overview
  DocumentUrl: https://developer.roku.com/docs/references/references-overview.md
```

File: manifests/r/Roku/RemoteTool/4.0.5/Roku.RemoteTool.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Roku.RemoteTool
+PackageVersion: 4.0.5
+PackageLocale: en-US
+Publisher: Roku
+PackageName: roku_remote_tool
```
Comment (mdanish-kh) [inline manifests/r/Roku/RemoteTool/4.0.5/Roku.RemoteTool.locale.en-US.yaml:8]:
```suggestion
PackageName: roku_remote_tool
PackageUrl: https://developer.roku.com/docs/features/features-overview.md
```

File: manifests/r/Roku/RemoteTool/4.0.5/Roku.RemoteTool.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+License: Proprietary
```
Comment (mdanish-kh) [inline manifests/r/Roku/RemoteTool/4.0.5/Roku.RemoteTool.locale.en-US.yaml:9]:
```suggestion
License: Apache-2.0
LicenseUrl: http://devtools.web.roku.com/RokuRemote/
```

File: manifests/r/Roku/RemoteTool/4.0.5/Roku.RemoteTool.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+Copyright: Copyright © 2022 Roku
```
Comment (mdanish-kh) [inline manifests/r/Roku/RemoteTool/4.0.5/Roku.RemoteTool.locale.en-US.yaml:10]:
```suggestion
Copyright: Copyright © 2022 Roku, Inc.
CopyrightUrl: http://devtools.web.roku.com/RokuRemote/
```

File: manifests/r/Roku/RemoteTool/4.0.5/Roku.RemoteTool.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Roku.RemoteTool
+PackageVersion: 4.0.5
+PackageLocale: en-US
+Publisher: Roku
```
Comment (mdanish-kh) [inline manifests/r/Roku/RemoteTool/4.0.5/Roku.RemoteTool.locale.en-US.yaml:7]:
```suggestion
Publisher: Roku
PublisherUrl: https://developer.roku.com/
PrivacyUrl: https://docs.roku.com/published/b2bprivacypolicy/en/us
PublisherSupportUrl: https://support.roku.com
```

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Review by mdanish-kh — APPROVED** [review]

**Review by mdanish-kh — APPROVED** [review]

**Pull request merged**